recipes-tag/xpdacq/meta.yaml

```diff
@@ -39,8 +39,8 @@ requirements:
 test:
   imports:
     - xpdacq
-    - xpdacq.xpdacq
-    - xpdacq.beamtime
+#    - xpdacq.xpdacq
+#    - xpdacq.beamtime
```

Comment:
ha, that will definitely help since it's failing on the tests.

Reply:
Exactly :)